### PR TITLE
Heroku config

### PR DESCRIPTION
**Pull request by dee-me-tree-or-love**
Configured for serving the app via [Heroku](https://volunteero-gates.herokuapp.com/)
Had to move a bunch (_admit, all_) of `devDependencies` to required. 
Configured so that the build script is run in production mode on deploy. 

It is live here:   
[https://volunteero-gates.herokuapp.com/](https://volunteero-gates.herokuapp.com/)

_Yet have to clean up the deps to keep only the stuff required for building and prune all the needless stuff._ 
☕️

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+// Used to start the site server
```
Comment (dmarkov00) [inline server.js:1]: It looks good. Mine with angular is almost the same: https://github.com/Volunteero/volunteero-front-end-base/blob/master/server.js.

File: package.json [hunk 2]
```diff
@@ -6,19 +6,19 @@
   "private": true,
   "scripts": {
     "dev": "webpack-dev-server --inline --progress --config build/webpack.dev.conf.js",
-    "start": "npm run dev",
-    "unit": "cross-env BABEL_ENV=test karma start test/unit/karma.conf.js --single-run",
+    "server": "node server.js",
+    "start": "npm run build && node server.js",
```
Comment (dmarkov00) [inline package.json]: I'm using the 'postinstall' keyword in my package.json, not that it matters, if you find it useful - https://github.com/Volunteero/volunteero-front-end-base/blob/master/package.json (just realized that I have some repetitive code there)

**Review by dmarkov00 — APPROVED** [review]
Nice!

File: package.json (hunk 2 above)
Comment (dmarkov00) [reply package.json]: My other comment didn't show up but it was about the server.js and that I have almost the same for the angular https://github.com/Volunteero/volunteero-front-end-base/blob/master/server.js. Looks good

Comment (dee-me-tree-or-love) [reply package.json]: Awesome! Thanks a lot dude, `postinstall` is indeed a nice idea! 
I will also take a look at how you handle dev dependencies in your app. 
This one ended up with a 85 MB package and I am not really happy with it - maybe any tips?

Thanks @dmarkov00  🤙

**Review by AlexProdrom — APPROVED** [review]
thumbs up

**Pull request merged**